src/mod/server.mod/servmsg.c

```diff
@@ -692,7 +692,8 @@ static int gotmsg(char *from, char *msg)
  */
 static int gotnotice(char *from, char *msg)
 {
-  char *to, *nick, ctcpbuf[512], *p, *p1, buf[512], *uhost = buf, *ctcp;
```

Comment:
@vanosg how does this work with ircv3 huge lines? shouldn't it be more than 512?

Reply:
I think we're OK here; I don't think wet pass the message tags (what increases it above 512) beyond the rawt function.